run_dnlm_flax.py

```diff
@@ -397,57 +341,50 @@ def write_eval_metric(summary_writer, eval_metrics, step):
     # )
 
     tokenizer = DebertaV2Tokenizer.from_pretrained(
-      model_args.tokenizer_name, 
-      unk_token="<unk>",
-      sep_token="<sep>",
-      pad_token="<pad>",
-      cls_token="",
-      mask_token="<mask>",
-      eos_token="<s>",
-      bos_token="</s>"
+        model_args.tokenizer_name,
+        unk_token="<unk>",
+        sep_token="</s>",
+        pad_token="<pad>",
+        cls_token="<s>",
```

Comment:
@morganmcg1 It's from the default config of `BartTokenizer`, so the tokenizer will add `<s>` and `</s>` tokens at the beginning and the end of the doc correctly. We can add it ourselves in the `SentenceTokenize` map() but it might be truncated by the tokenizer

Reply:
the cls_token here only act as a bos token for the encoder

Reply:
ah ok, makes sense 👍